test/comparisons.jl

```diff
@@ -406,6 +406,15 @@ const INTERVAL_TYPES = [Interval, AnchoredInterval{Ending}, AnchoredInterval{Beg
         @test superset([a, b]) == Interval(1, 5, true, true)
     end
 
+    @testset "equal -0.0/0.0" begin
+        a = Interval(0.0, -0.0)
+        b = Interval(-0.0, 0.0)
+
+        @test a == b
+        @test !isequal(a, b)
+        @test hash(a) != hash(b)
```

Comment:
Should add the other comparisons to ensure they work as expected

Reply:
The endpoint comparisons?

Reply:
The other comparisons which are performed in this file like `<` and so on